### ISSUE TITLE
Readme doesn't show initial config

### ISSUE THREAD

**Issue by kurtcarpenter** [gt-big-data/QDoc#23]
Add something about the cloneCollection() commands needed.

Also, the mongodb path information is outdated as of 3.2

**Comment by supersam654** [comment]
Exact mongo commands are:

```
mongo
use big_data
db.feed.drop()
db.test_sources.drop()
db.cloneCollection(‘db.retinanews.net’, ‘feed’)
db.cloneCollection(‘db.retinanews.net’, ‘test_sources’)
```

Those, along with info about requiring Mongo 3.2 (not just 3), belong in the README.

**Comment by kurtcarpenter** [comment]
Fixed in PR #25
